pytket/binders/circuit/boxes.cpp

```diff
@@ -186,9 +205,16 @@ void init_boxes(py::module &m) {
       "tensor of Pauli operations and their (possibly symbolic) phase "
       "parameters.")
       .def(
-          py::init<
-              const std::vector<std::pair<std::vector<Pauli>, Expr>> &,
-              CXConfigType>(),
+          py::init([](const std::vector<std::pair<std::vector<Pauli>, Expr>>
+                          &py_gadgets,
```

Comment:
This change is left over from a change that I found out was no longer necessary. It makes no sense anymore. I'll remove it